frontend/ui/src/app/components/side-bar/side-bar.component.ts

```diff
@@ -55,10 +59,15 @@ export class SideBarComponent {
   protected readonly faAddressBook = faAddressBook;
   protected readonly faBox = faBox;
   protected readonly buildConfig = buildConfig;
+  protected readonly faArrowRightLong = faArrowRightLong;
+  protected readonly faHome = faHome;
+  protected readonly faChevronDown = faChevronDown;
 
   @ViewChild('asideElement') private asideElement?: ElementRef<HTMLElement>;
-
-  protected readonly artifactsSubMenuOpen = signal(true);
+  protected readonly agentsSubMenuOpen = signal(true);
+  protected readonly licenseSubMenuOpen = signal(true);
+  protected readonly registrySubMenuOpen = signal(true);
+  protected readonly licenseOverlayOpen = signal(false);
```

Comment:
Maybe something for later, but would it make sense to only have 1-2 submenus open initially and maybe persist the state in local/session storage?